cluster/confighash.go

```diff
@@ -0,0 +1,129 @@
+// Copyright © 2022 Obol Labs Inc.
+//
+// This program is free software: you can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the Free
+// Software Foundation, either version 3 of the License, or (at your option)
+// any later version.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of  MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+// more details.
+//
+// You should have received a copy of the GNU General Public License along with
+// this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package cluster
+
+import ssz "github.com/ferranbt/fastssz"
+
+// staticDefinition defines the static (non-changing) portion of the charon cluster definition.
+type staticDefinition struct {
```

Comment:
needed to create this as this seems the only easy way to derive ssz hashes on. I wanted to extract the common fields of the definition object, but i hope a little duplication is okay?

Reply:
yeah, the fields of this struct can be private though